helpdesk_mgmt_stage_server_action/models/helpdesk_ticket_stage.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HelpdeskTicketStage(models.Model):
+    _inherit = "helpdesk.ticket.stage"
+
+    action_id = fields.Many2one("ir.actions.server", string="Server Action")
```

Comment:
Please add a help section to explain how the field is used, and which values in passed in context of the action (on model `helpdesk.ticket` and with ticket ids).

Also I would have a restriction here to have only actions with `model_id` = `helpdesk.ticket`.